public/manifest.webmanifest

```diff
@@ -4,7 +4,7 @@
   "icons": [
     {
       "src": "img/favicon/favicon.ico?v=[[VERSION]]",
-      "sizes": "64x64",
+      "sizes": "48x48",
```

Comment:
Can you elaborate on this change?

Reply:
The favicon `public/img/favicon/favicon.ico` has resolution 48x48. Chrome DevTools report an error if we declare it as 64x64 resolution. It is not a crucial change (the PWA is still installable by Chrome), but I just noticed the error and thought I'd fix it.

![resolution_error](https://user-images.githubusercontent.com/24862276/76862125-e5827200-685d-11ea-872b-05f837cb3692.png)

Reply:
Makes sense. Good catch. :slightly_smiling_face: